src/components/validation/sync.rs

```diff
@@ -233,27 +235,31 @@ impl TaskCoordinator<Configuring> {
                 tasks_completed: Some(tasks_completed),
                 shutdown_triggers: shutdown_triggers.into_inner().expect("poisoned"),
             },
+            name: self.name,
         }
     }
 }
 
 impl TaskCoordinator<Started> {
     /// Triggers all coordinated tasks to shutdown, and waits for them to mark themselves as completed.
-    pub async fn shutdown(mut self) {
+    pub async fn shutdown(&mut self) {
+        info!("Triggering {} task to shutdown.", self.name);
```

Comment:
FWIW, you might consider a logging format of `{name}: <message here>` so that the task name is always the prefix. Easier for filtering/grepping logs.